lib/game_setup.rb

```diff
@@ -0,0 +1,112 @@
+class GameSetup
+  def initialize
+    @token1 = ''
+    @token2 = ''
+    @size = 3
+    @mode = :computerVScomputer
+    @order = :player1
+    @board_size = {'1' => 3, '2' => 4}
+    @modes = {'1' => :humanVShuman, '2' => :humanVScomputer, '3' => :computerVScomputer}
+    @orders = {'1' => :player1, '2' => :player2}
```

Comment:
modes, orders, and board_sizes don't appear to fit the bill of "instance" variable. They feel permanent. Let's refactor that a little bit.

Reply:
I made these class variables.  Debated placing them in the methods since they are really only used there, but I liked them more visible.  Let me know what you think.